supertokens_python/recipe/session/framework/fastapi/__init__.py

```diff
@@ -41,7 +42,8 @@ def verify_session(
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[..., Coroutine[Any, Any, Union[SessionContainer, None]]]:
     _ = user_context
-    from fastapi import Request
+
+    from fastapi import Request  # pylint: disable=reimported
```

Comment:
why this `# pylint: disable=reimported`? Just remove the import from top?

Reply:
Fixed.